lib/mpool/src/mblock_fset.c

```diff
@@ -51,7 +51,7 @@ struct mblock_fset {
     struct media_class  *mc;
     struct kmem_cache   *rmcache[MBLOCK_FSET_RMCACHE_CNT];
 
-    atomic64_t           fidx;
+    atomic_int           fidx;
```

Comment:
This used to be an 64-bit atomic type, so atomic_ulong.

Reply:
Agreed it should be unsigned so that it doesn't go wonky when it rolls over in mblock_fset_alloc().  But there you assign it to an "int", which probably should be a uint, right?  And fcnt should be a uint too???

Reply:
Nevermind, ulong should be fine...